gs-element-blockly.html

```diff
@@ -286,8 +294,24 @@ <h2>Hello gs-element-blockly</h2>
           }
           return;
         }
-        let toolbox = this._createToolbox();
-        this.workspace.updateToolbox(toolbox);
+        this._updateToolbox(this._createToolbox());
+      },
+
+      _updateToolbox(toolboxXml) {
+        // Utilizado solamente para testear
+        this._toolboxXml = toolboxXml;
+
+        try {
+          this.workspace.updateToolbox(toolboxXml);
+        } catch (error) {
+          // Blockly no soporta cambiar del modo sin categorías al modo con, ni viceversa.
```

Comment:
Spanglish in this repo predates your PR, doesn't it?

Reply:
Yep. _Ya estaba así desde antes._ Comments are mostly in spanish, so I tried to honor that.